docs/source/graph-construction/gs-processing/example.rst

```diff
@@ -281,28 +290,72 @@ in an ``edge_data`` directory.
     for node id 1 etc.
 
 
-At this point you can use the DGL distributed partitioning pipeline
-to partition your data, as described in the
-`DGL documentation <https://docs.dgl.ai/guide/distributed-preprocessing.html#distributed-graph-partitioning-pipeline>`_
-.
+Run a GPartition job locally
+---------------------------
+While :ref:`GPartition<gpartition_index>` is designed to run on a multi-machine cluster,
+you can run GPartition job locally for the example. Once you have completed previous installation
+and GSProcessing example, you can proceed to run the GPartition example.
+
+Assuming your working directory is ``graphstorm``,
+you can use the following command to run the processing job locally:
+
+.. code:: bash
 
-To simplify the process of partitioning and training, without the need
-to manage your own infrastructure, we recommend using GraphStorm's
-`SageMaker wrappers <https://graphstorm.readthedocs.io/en/latest/scale/sagemaker.html>`_
-that do all the hard work for you and allow
-you to focus on model development. In particular you can follow the GraphStorm documentation to run
-`distributed partitioning on SageMaker <https://github.com/awslabs/graphstorm/tree/main/sagemaker#launch-graph-partitioning-task>`_.
+    echo 127.0.0.1 > ip_list.txt
+    python3 -m graphstorm.gpartition.dist_partition_graph \
+        --input-path /tmp/gsprocessing-example/ \
+        --metadata-filename updated_row_counts_metadata.json \
+        --output-path /tmp/gpartition-example/ \
+        --num-parts 1 \
+        --dgl-tool-path ./dgl/tools \
+        --partition-algorithm random \
+        --ip-config ip_list.txt \
+        --do-dispatch
 
+Examining the job output of GPartition
+--------------------------------------
+
+Once the partition job is done, you can examine the outputs it has created.
 
-To run GSProcessing jobs on Amazon SageMaker we'll need to follow
-:ref:`GSProcessing distributed setup<gsprocessing_distributed_setup>` to set up our environment
-and :ref:`Running GSProcessing on SageMaker<gsprocessing_sagemaker>` to execute the job.
+.. code-block:: bash
 
+    $ cd /tmp/gpartition-example
+    $ ls -ltR
+
+    dist_graph/
+        metadata.json
+        |- part0/
+            edge_feat.dgl
+            graph.dgl
+            node_feat.dgl
+            orig_eids.dgl
+            orig_nids.dgl
+    partition_assignment/
+        director.txt
+        genre.txt
+        movie.txt
+        partition_meta.json
+        user.txt
+
+Now there are a few directories containing partitioned graph ready for training and inference.
+
+* ``metadata.json``: This file contains metadata about the distributed DGL graph. The files under
+``dist_graph`` should be ready for the training/inference pipeline.
```

Comment:
Should the ``dist_graph`` folder be introduced first before talking about the ``metadata.json``?

And,

May change to "The folder ``dist_graph`` contains files ready for GraphStorm training and inference, including:"